src/components/sections/jobs/JobPostingList.tsx

```diff
@@ -0,0 +1,81 @@
+"use client";
+
+import { useEffect, useMemo, useState } from "react";
+
+import JobPosting from "src/components/jobPosting/JobPosting";
+import { Tag } from "src/components/tag";
+import Text from "src/components/text/Text";
+import { CompanyLocation } from "studio/lib/interfaces/companyDetails";
+import { IJobPosting } from "studio/lib/interfaces/jobPosting";
+
+import styles from "./jobs.module.css";
+
+interface JobPostingListProps {
+  jobPostings: IJobPosting[];
+  companyLocations: CompanyLocation[];
+}
+
+export default function JobPostingList({
+  jobPostings,
+  companyLocations,
+}: JobPostingListProps) {
+  const [locationFilter, setLocationFilter] = useState<CompanyLocation | null>(
+    null,
+  );
+  const [filteredJobPostings, setFilteredJobPostings] =
+    useState<IJobPosting[]>(jobPostings);
+  const jobPostingLocations = useMemo(
+    () =>
+      Object.fromEntries(
+        jobPostings.map((jobPosting) => [
+          jobPosting._key,
+          jobPosting.locations.map((location) => location.companyLocationName),
+        ]),
+      ),
+    [jobPostings],
+  );
+
+  useEffect(() => {
+    if (locationFilter === null) {
+      setFilteredJobPostings(jobPostings);
+    } else {
+      setFilteredJobPostings(
+        jobPostings?.filter((jobPosting) =>
+          jobPostingLocations[jobPosting._key].includes(
+            locationFilter.companyLocationName,
+          ),
+        ),
+      );
+    }
+  }, [locationFilter, jobPostings, jobPostingLocations]);
+
+  return (
+    <div className={styles.jobPostingsContainer}>
+      <div className={styles.filters}>
+        <Text type="labelRegular">Kontor</Text>
+        <Tag
+          active={!locationFilter}
+          type="button"
+          onClick={() => setLocationFilter(null)}
+          text="Alle"
```

Comment:
You need a translation for the text in the Tag, we already have one defined for the word "all", so if you add `const t = await getTranslations("employee_card");` and `text=t("all")` we should be able to access all translations of the object 🤓

Reply:
Will fix in a separat PR!